pyvista/plotting/widgets.py

```diff
@@ -2629,6 +2633,77 @@ def clear_button_widgets(self):
         """Remove all of the button widgets."""
         self.button_widgets.clear()
 
+    def add_logo_widget(
+        self,
+        logo: Optional[Union[pyvista.ImageData, str, pathlib.Path]] = None,
+        position: Tuple[float, float] = (0.75, 0.8),
+        size: Tuple[float, float] = (0.2, 0.2),
+        opacity: float = 1.0,
+    ):
+        """Add a logo widget to the top of the viewport.
+
```

Comment:
We should also add a note about default behavior, either in the description here like in the theme, or in the parameter description.

Reply:
Done in ba04807cd2f02ff8d485612484c0061aa969913a